lib/core/src/server/manager/manager-webpack.config.js

```diff
@@ -0,0 +1,80 @@
+import webpack from 'webpack';
+import Dotenv from 'dotenv-webpack';
+import HtmlWebpackPlugin from 'html-webpack-plugin';
+import HtmlWebpackHarddiskPlugin from '@ndelangen/html-webpack-harddisk-plugin';
+import CaseSensitivePathsPlugin from 'case-sensitive-paths-webpack-plugin';
+
+import { version } from '../../../package.json';
+import { getManagerHeadHtml } from '../utils/template';
+import { excludePaths, loadEnv, getBabelRuntimePath } from '../config/utils';
+
+export default ({ configDir, entries, outputDir }) => {
+  const { raw, stringified } = loadEnv();
+
+  return {
+    name: 'manager',
+    mode: 'production',
+    bail: true,
+    devtool: 'none',
+    entry: entries,
+    output: {
+      path: outputDir,
+      filename: '[name].[chunkhash].bundle.js',
+      publicPath: '',
+    },
+    plugins: [
+      new HtmlWebpackPlugin({
+        filename: `index.html`,
+        chunksSortMode: 'none',
+        alwaysWriteToDisk: true,
+        inject: false,
+        templateParameters: (compilation, files, options) => ({
+          compilation,
+          files,
+          options,
+          version,
+          headHtmlSnippet: getManagerHeadHtml(configDir, process.env),
+        }),
+        template: require.resolve(`../templates/index.ejs`),
+      }),
+      new HtmlWebpackHarddiskPlugin(),
+      new webpack.DefinePlugin({ 'process.env': stringified }),
+      new CaseSensitivePathsPlugin(),
+      new Dotenv({ silent: true }),
+    ],
+    module: {
+      rules: [
+        {
+          test: /\.jsx?$/,
```

Comment:
Assuming we have a manager client code already transpiled, we can remove this rule, right?

Reply:
Likely, but users could load local un-compiled addons, in addons.js

Reply:
😞

Reply:
Should we even handle this usecase ?

Reply:
I think we should, it makes it easier to get started writing addons?